config/confd/templates/server.conf.tmpl

```diff
@@ -57,3 +57,8 @@ tcp-nodelay
 # disable tls renegotiation hourly in favour of every 1GB to avoid load spikes
 reneg-sec 0
 reneg-bytes 1073741824
+
+{{ if getenv "VPN_UDP_SERVER_EXPLICIT_EXIT_NOTIFY" }}
+# send RESTART control channel command to connected clients
+{{ getenv "VPN_UDP_SERVER_EXPLICIT_EXIT_NOTIFY" }}
```

Comment:
What is meant to be in here? This seems like it could contain any openvpn config?

Reply:
@Page- it is meant for controlling this setting:

```
--explicit-exit-notify n
--
  | In UDP client mode or point-to-point mode, send server/peer an exit notification if tunnel is restarted or OpenVPN process is exited. In client mode, on exit/restart, this option will tell the server to immediately close its client instance object rather than waiting for a timeout.The n parameter (default 1 if not present) controls the maximum number of attempts that the client will try to resend the exit notification message.In UDP server mode, send RESTART control channel command to connected clients. The n parameter (default 1 if not present) controls client behavior. With n = 1 client will attempt to reconnect to the same server, with n = 2 client will advance to the next server.OpenVPN will not send any exit notifications unless this option is enabled.
```

I suppose we can just use this parameter for specifying `n` only, so it would become:

```
{{ if getenv "VPN_UDP_SERVER_EXPLICIT_EXIT_NOTIFY" }}
# send RESTART control channel command to connected clients
explicit-exit-notify {{ getenv "VPN_UDP_SERVER_EXPLICIT_EXIT_NOTIFY" }}
{{ end }}
```